src/index.html

```diff
@@ -7,8 +7,39 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+
+      <a href="#home">
+        <img href="#home" 
+          class="header__logo--img" 
+          src="images/logo.png" 
+          alt="MOYO Logo">
+      </a>
+
+      <nav class="header__nav">
+        <ul class="nav__list">
+          <li><a href="#apple" class="nav__link is-active">Apple</a></li>
+          <li><a href="#samsung" class="nav__link">Samsung</a></li>
+          <li><a href="#smartphones" class="nav__link">Smartphones</a></li>
+          <li>
+            <a href="#laptopscomputers" 
+              class="nav__link"
+              data-qa="hover">Laptops & Computers</a>
+          </li>
+          <li><a href="#gadgets" class="nav__link">Gadgets</a></li>
```

Comment:
Please refer to this [guideline](https://mate-academy.github.io/style-guides/htmlcss.html#html-line-wrapping) and fix the formatting